### ISSUE TITLE
manual peak picking with `optimize=true` should also return width

### ISSUE THREAD

**Issue by kjappelbaum** [cheminfo/common-spectrum#63]
i think a more natural behavior of `optimize=true` would be to also fit a shape to make it consistent with the automatic peak picking. This is also important for many techniques such as XPS or XRD where we actually also want to know the width of the peak

**Comment by lpatiny** [comment]
Currently we give back the width after optimizePeaks: https://github.com/mljs/global-spectral-deconvolution/blob/81f286b6cd6fb54e48a25731a95e20c3ec739b6e/src/post/__tests__/optimizePeaks.test.js#L21-L29

But I guess we would need to provide also the other optimization parameters if there are some. @kjappelbaum what would you like to have ?

**Comment by kjappelbaum** [comment]
> The width in the manual peak picking result ? I fully agree.

yes, i'd like to have the width in the manual peak picking to have an output that is interoperable with the one from automatic picking
